core/src/main/java/com/google/zxing/ResultMetadataType.java

```diff
@@ -93,5 +93,15 @@
    * parity is given with it.
    */
   STRUCTURED_APPEND_PARITY,
-  
+
+  /**
+   * For Code39 this contains the decoded contents with the Full-ASCII range.
+   */
+  CODE_39_EXTENDED,
```

Comment:
Call this "CODE_39_FULL_ASCII" ? IIRC there is another format mode called 'extended' but I can't find it now.